.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+**Description:**
+<!--- What does this PR do? -->
+
+**Motivation and context:**
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+**How has this been tested?**
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Reviewers will test your PR in at least this way. -->
+
+**Where should a reviewer start?**
+<!-- Indicate the best part to start looking to understand the changes. -->
+
+**Types of changes:**
+<!--- What types of changes does your code introduce? -->
+<!-- Put an `x` in all the boxes that apply: -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that causes existing functionality to change)
+
+**Checklist:**
+<!--- Go over all the following points. Put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. -->
+<!--- We're here to help! -->
+
+- [ ] I have read the **CONTRIBUTING.rst** document.
```

Comment:
Is every contributor going to have to fill out these check boxes every time they make a pull request? I would find this a bit annoying.

If yes, I would reduce them to:
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] My change requires a changelog entry and I have included a changelog entry.
- [ ] I have added tests to cover my changes.